bin/demeteorizer

```diff
@@ -42,7 +48,9 @@ var options = {
   input: process.cwd(),
   directory: Program.output,
   json: Program.json,
-  architecture: Program.architecture
+  architecture: Program.architecture,
+  verbose: Boolean(Program.verbose),
+  server: Program.server || "localhost:3000"
```

Comment:
Replace the double quotes with single quotes; remove the `server` default value in `lib/build.js`; update the help to reflect the default (add `[localhost:3000]` to the description); and get the coverage back up to 100% so the tests pass and it's good.